.gitignore

```diff
@@ -65,4 +65,5 @@ notebooks/.ipynb_checkpoints
 .jenkins*
 cfme-qe-yamls
 rhci-qe-data
+artifacts/artifactor.log
```

Comment:
I think this is specific to your setup, my artifactor logs into $ROOT/log so this is not relevant for everyone, can we please remove it.